myosuite/envs/myo/myochallenge/run_track_v0.py

```diff
@@ -272,10 +275,12 @@ def _maybe_sample_terrain(self):
             self.sim.model.geom_pos[self.sim.model.geom_name2id('terrain')] = np.array([0, 0, -10])
 
     def _randomize_position_orientation(self, qpos, qvel):
```

Comment:
at some point we should do this only with quaternion operations, but it's okay for now